Archipelago.HollowKnight/Archipelago.cs

```diff
@@ -275,6 +478,49 @@ private LoginResult ConnectToArchipelago()
             }
         }
 
+        private void OnDeathLinkReceived(DeathLink deathLink)
+        {
+            if (outgoingDeathlinks > 0)
+            {
+                outgoingDeathlinks--;
+                return;
+            }
+            if (deathlinkStatus == DeathLinkStatus.None)
+            {
+                deathlinkStatus = DeathLinkStatus.Pending;
+            }
+            string cause = deathLink.Cause;
+            if (cause == null || cause == "")
+            {
+                cause = $"{deathLink.Source} died.";
+            }
+            new MsgUIDef()
+            {
+                name = new BoxedString(cause),
+                sprite = new BoxedSprite(DeathLinkSprite)
+//                    GameCameras.instance.hudCamera.GetComponentsInChildren<JournalEntryStats>(true)
+//                    .Where(x => x.notesConvo == "NOTE_HOLLOW_SHADE")
+//                    .First()
+//                    .GetSprite()
+//                )
+            }.SendMessage(MessageType.Corner, null);
+/*
+            ArchipelagoLocation location = new(deathLink.Source);
+            AbstractPlacement pmt = location.Wrap();
+            pmt.Add(new DeathlinkItem(deathLink.Source, deathLink.Cause));
+            var tag = pmt.AddTag<InteropTag>();
+            tag.Properties["DisplaySource"] = deathLink.Source;
+            ItemChangerMod.AddPlacements(pmt.Yield());
+            pmt.GiveAll(RemoteGiveInfo);
+*/
```

Comment:
Remove comments if unnecessary.

Reply:
This is a TODO.  I want deaths to also show in the RecentItems list, but when implemented that way I was getting weird issues when loading/saving games (ItemChanger would never unhook and thus multiple connections to AP would end up being made.)